analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -65,6 +65,23 @@ def parse_checkers(clangsa_output):
     return checkers_list
 
 
+def get_version_info(analyzer_binary, environ=None):
```

Comment:
In #2398 I've put the code to get the version under the clangsa module.

The code to get the clang version is at many places in the code I think we should put it to one place and use only that.

```
analyzer/codechecker_analyzer/host_check.py
27:    clang_version_cmd = [compiler_bin, '--version']

analyzer/codechecker_analyzer/analyzer.py
78:        version = [analyzer_bin, u' --version']

analyzer/codechecker_analyzer/cmd/analyzers.py
146:                                                   '--version'])

analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py
67:            self.__analyzer_binary, ['--version'], self.environ)

analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py
120:                [analyzer_binary, '--version'],
```